packages/loader/container-loader/src/loader.ts

```diff
@@ -128,7 +128,8 @@ function createCachedResolver(resolver: IUrlResolver) {
     return cacheResolver;
 }
 
-export interface ILoaderOptions extends ILoaderOptions1{
```

Comment:
my goal was actually the opposite here. i think we should be moving all the config here, rather than container definitions, as i don't think there is a need for loader configs to be in container definitions as they are not a layer crossing interface.

Reply:
I'll revert this, it seems orthogonal to the work